tests/test_strategy_one.py

```diff
@@ -1,30 +1,30 @@
 #!/usr/bin/env python
 import unittest
-from game.player_with_strategy_one import PlayerWithStrategyOne
+from game.player import Player
 from game.gameboard import GameBoard
+from game.strategy_one import StrategyOne
 
 class TestPlayer(unittest.TestCase):
     def setUp(self):
-        self.player_blue = PlayerWithStrategyOne('blue')
-        self.player_red = PlayerWithStrategyOne('red')
+        self.player_blue = Player('blue', StrategyOne())
```

Comment:
When you refactor a thing like a strategy out of a class hierarchy you need to do more than just tweaking the test so it passes again. In this case you need to refactor the test. This test is currently not a unit test on the strategy, as you are hitting the `Player` instance,  not the `StrategyOne`. Remember that we discussed many times the mapping between the test class and the model class (hint: you should test the `return_column` method).